### DISCUSSION TITLE
Toyota Connect HA for Australian users

### DISCUSSION THREAD

**Discussion by lynchpinhags** [DurgNomis-drol/ha_toyota#149] [Q&A]
Hi all,
I recently started using Home Assistant, and was very keen to try and get my Toyota app info into HA. I'm in Australia though, and I just wondered if there was any scope with this project for it to work with Australian users too? Or is our implementation of the Toyota Connect app etc totally different? Happy to help with testing etc if needed, I'm not a coder so unfortunately don't have the skillset to contribute myself really! Cheers.

**Comment by DurgNomis-drol** [comment]
Not possible as this only works for EU sorry.
